var/spack/repos/builtin/packages/mpix-launch-swift/package.py

```diff
@@ -23,10 +23,9 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
-from distutils.dir_util import copy_tree
 
 
-class MpixLaunchSwift(Package):
+class MpixLaunchSwift(MakefilePackage):
```

Comment:
again not sure if this should be wrapped up in this PR, but seems innocent enough